packages/front-end/components/Features/RuleModal/index.tsx

```diff
@@ -113,6 +115,11 @@ export default function RuleModal({
     ruleType: defaultType,
     attributeSchema,
   });
+
+  // unset the ID if we're duplicating the rule.
+  if (duplicate) {
+    rule.id = "";
```

Comment:
This mutates the original object and should not be done in the main render loop. Better to just modify `values` when submitting instead.

Reply:
done